Styles/flexbox.css

```diff
@@ -77,8 +77,7 @@ html, body {
 	}
 	#brdf-canvas {
 		width: 80vw;
-		height: 80vw;
-
+		height: 68.5714286vw;
```

Comment:
68.5714286 is a "magic number" that scares me a bit. Why exactly did we choose this number? There should at least be a comment.